google/cloud/spanner_dbapi/connection.py

```diff
@@ -124,6 +125,24 @@ def spanner_client(self):
         """
         return self._instance._client
 
+    @property
+    def schema_name(self):
+        """schema name for this database.
```

Comment:
This property indicates the default schema that is used by this __connection__, not the database that it is connected to.

```suggestion
        """schema name for this connection.
```

Reply:
Done